### ISSUE TITLE
Regarding the calculation of GWD

### ISSUE THREAD

**Issue by Vayne0227** [uzh-rpg/event_representation_study#1]
Hello author, if I only want to calculate the GWD value of the voxel grid, can I just run "gromov_wasserstein. py"

**Comment by NikolaZubic** [comment]
Hi, this is the core computation file with an example inside `if __name__ == "__main__".` Here, we showed in the toy example that increasing the number of channels leads to a decrease in GWD, which makes sense since we accumulate more information in the representation - the discrepancy is becoming lower. You can use this code for any dataset or representation.

**Comment by Vayne0227** [comment]
Thank you for your answer, but I have a few more questions regarding the issues encountered when running gen1_computer.py

1.When using the "TimeSurface" event representation, the observed GWD value is 0.3696, which appears to differ from the value reported in the paper.

2.When the event representation is set to "TORE," I noticed that the program has been running for an extended period without completion. Could you please confirm if this behavior is expected or if there might be any potential issues causing the prolonged execution?

**Comment by NikolaZubic** [comment]
1. Make sure you have the same environment installation and seeds as we had, also do not change the code (like you did with ERGO-12, making it a one-channel representation), and it should be the same/similar, at least the ordering.

2. Regarding the TORE representation, it was not optimized as we just wanted to compare how it performs in terms of mAP - GWD correlation. Since it was not the best-performing representation, we did not invest time optimizing it.
I just took MATLAB code and wrote Python script as fast as possible. We do not have a plan to write a more optimized version of TORE as it was not the main focus of the paper.
